readme.txt

```diff
@@ -16,6 +16,14 @@ This provides a way of scheduling recurring meetings, and displaying a calendar
 
 == Installation ==
 
+1.	Make sure you have [`git`](https://git-scm.com/), [`node`](https://nodejs.org/), and [`npm`](https://www.npmjs.com/get-npm) installed.
+2.	Clone this repository into your `\plugins` folder.
+3.	Execute `npm install` from the root directory of the repository to install the dependencies.
+4.	Execute `npm start` for development mode (`npm run build` for a production build).
```

Comment:
Users downloading this from wp.org shouldn't have to do these steps, we should publish a built package. So just 5-7 should be fine 🙂

Reply:
👍Good call